rootfs/rootfs/etc/rc.d/vmtoolsd

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+/usr/local/bin/vmtoolsd --background /var/run/vmtoolsd.pid
```

Comment:
I feel like I've asked this before, but is there a nice easy way we can make this conditional based on whether we're really in VMware without hacky scraping of `/sys` or `/proc` or relying on the `vmtoolsd` binary to "be a good citizen"?  For example, both the VirtualBox-specific code and the HyperV-specific code have an `if modeprobe some-module ...` that works really, really well to make sure we don't even attempt to run the code unless we've got the necessary hardware we expect.